### PR TITLE
Des 674

### PR DESCRIPTION
**Pull request by StommePoes**

### CODE REVIEW

File: docs/elements/inputs.md [hunk 1]
```diff
@@ -4,172 +4,193 @@ title: Inputs
...
-  <label class="pe-textLabelInput__label" for="z">Label</label>
-  <input type="text" class="pe-textInput--basic" id="z" placeholder="Placeholder">
+  <label class="pe-textLabelInput__label" for="anInput">Label</label>
+  <input type="text" class="pe-textInput--basic" id="anInput" aria-describedby="someHint">
+ <span id="someHint" class="pe-input--info_message">this is an info message</span>
 {{/demo}}
 
 ## Inputs - Basic (single line - focus)
 
+This is just showing that, on focus, the label should turn blue. This needs JavaScript to do so is not naturally included in Elements css.
```
Comment (mhomolak) [inline docs/elements/inputs.md]: > This needs JavaScript to do so is not naturally included in Elements css.

and is not naturally .... etc.?

File: docs/elements/inputs.md [hunk 2]
```diff
@@ -238,140 +259,138 @@ Select dropdowns are generally meant for times when a user needs to make a singl
...
-<textarea class="pe-multiLineText--readOnly" id="zz" cols="30" rows="5"
-  placeholder="Many words displayed over multiple lines." disabled>
-</textarea>
+<label class="pe-textLabelInput__label" for="andale">Multi-line label</label>
+<textarea class="pe-multiLineText--readOnly" id="andale" cols="30" rows="5" readonly>TERMS AND CONDITIONS. You ("you") agree to giving up an arm, a leg, your first born, and your dawg. These terms may be changed at any time without warning.
+3. TRANSFER. You may not rent, lease, lend, sell, redistribute, or sublicense the iOS software. You may, however (a)
+Oh you know what? This is page 46, nobody's still reading this. I bet only about five people clicked to read the T&amp;Cs in the first place - we might as well just say anything we like.
+Tony on floor 5 of Apple HQ smells of sardines.
+When someone sends a funny email around the offices we have to reply with iLaughed. It's in our job description.
+Remember that legal kerfuffle over Apple &amp; Apple studios? Want to know how we fixed it? We bought The Beatles. We have the surviving ones come and sing to us for scraps. We're looking at ways to reanimate the dead ones.
```
Comment (mhomolak) [inline docs/elements/inputs.md]: I had no idea!!

File: docs/elements/inputs.md (hunk 1 above)
Comment (StommePoes) [inline docs/elements/inputs.md]: so -> and?  I can do that.

Comment (StommePoes) [inline docs/elements/inputs.md]: weird, the change I see locally but not here.

**Comment by umahaea** [comment]
Regression passed:, so good to merge.
Desktop: https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/234149943
Mobile: https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/234157572

**Pull request merged**